toolkit/tools/internal/customizationmacros/customizationmacros.go

```diff
@@ -0,0 +1,133 @@
+// Copyright Microsoft Corporation.
+// Licensed under the MIT License.
+
+/*
+customizationmacros package provides functions to add customization macros to a root directory. These macros are saved in the
+root directory's default rpm macros directory. Each customization has a corresponding macro file that is generated as needed.
+*/
+package customizationmacros
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/file"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/logger"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/rpm"
+)
+
+const (
+	// macro files to customize the installer and final images
+	DisableDocsMacroFile         = "macros.installercustomizations_disable_docs"
+	ConfigureRpmLocalesMacroFile = "macros.installercustomizations_customize_locales"
```

Comment:
Do we need these to be global (and public)? Seems like we only need the constants in one function.

Reply:
Feel free to make them private. If MIC needs them, then it isn't too difficult to make them public again.

Reply:
Private it is, I'll leave them as a const up top since that makes more sense to me.